docs/readme.md

```diff
@@ -1,15 +1,17 @@
 # Deploying Azure Industrial IoT
 
-Deploying the services includes deploying the Azure Industrial IoT micro services to Azure as well as the corresponding modules to Azure IoT Edge.  The following articles provide more information how to...
+Deploying Azure Industrial IoT includes deploying the Azure Industrial IoT Microservices to Azure in addition to its dependencies and the corresponding modules to Azure IoT Edge. The following articles provide more information about how to deploy both Microservices and edge modules in addition to the dependencies.
```

Comment:
"their dependencies" since reflexive to Microservices, not IoT.  Also - reduandency with the second sentence.  There - also "their dependencies".